recipes/jasper/all/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_MODULE_PATH ${CONAN_CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/source_subfolder/build/cmake/modules ${CMAKE_MODULE_PATH})
```

Comment:
Why is this needed?

Reply:
Because in its CMakeLists, Jasper use CMAKE_SOURCE_DIR instead of CMAKE_CURRENT_SOURCE_DIR
https://github.com/mdadams/jasper/blob/9aef6d91a82a8a6aecb575cbee57f74470603cc2/CMakeLists.txt#L6

Reply:
Thanks for the clarification!